x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_details_page/components/agent_logs/agent_logs.tsx

```diff
@@ -119,6 +121,11 @@ export const AgentLogsUI: React.FunctionComponent<AgentLogsProps> = memo(
   ({ agent, agentPolicy, state }) => {
     const { data, application, http } = useStartServices();
     const { update: updateState } = AgentLogsUrlStateHelper.useTransitions();
+    const config = useConfig();
+    // console.log(JSON.stringify(config, null, 2));
+    const isLogsViewAvailable =
+      !config.internal?.registry?.capabilities ||
+      config.internal?.registry?.capabilities?.includes('observability');
```

Comment:
we want to show logs ui for observability projects and stateful (which doesn't set capabilities)